src/http.rs

```diff
@@ -252,6 +245,245 @@ impl HttpClient {
     }
 }
 
+#[cfg(feature = "async")]
+use futures::prelude::*;
+
+#[cfg(feature = "async")]
+impl HttpClient {
+    pub async fn tx_log(&self, actions: Vec<Action>) -> impl Future<Output = TxLogResponse> + Send {
+        let actions_str = actions
+            .into_iter()
+            .map(|edn| edn.serialize())
+            .collect::<Vec<String>>()
+            .join(", ");
+        let mut s = String::new();
+        s.push_str("[");
+        s.push_str(&actions_str);
+        s.push_str("]");
+
+        let resp = self
+            .client
+            .post(&format!("{}/tx-log", self.uri))
+            .headers(self.headers.clone())
+            .body(s)
+            .send()
+            .await
+            .unwrap()
```

Comment:
Is this correct, should we have this many unwraps?

Reply:
It comes from this problem `Future<Output = Result<T, CruxError>>` :(

Reply:
Okay, as we discussed outside of here, we will do Results in another PR, and add on README.md that the async feature is beta.